.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules/
 **/**/.DS_Store
 .env
+package-lock.json
```

Comment:
Package Lock needs to be in the file or else it won't load the necessary node packages

Reply:
I think I deleted it already from the pull req. But if that didn't work, I think you can just approve everything from the pull request besides the .gitignore and then delete it from the que